content/md/assets/upload-assets_fr

```diff
@@ -0,0 +1,46 @@
+---
+id: upload-assets
+themes: manage-your-assets, import-export-data
+title: "**Upload** assets"
+popular: false
+ee-only: true
+related: work-with-assets, assets-transformation
+---
+
+
+# Aperçu
+Le chargement en masse des ressources numériques est particulièrement utile si vous voulez mettre à jour votre catalogue avec des sources externes: shooting photo, nouvelle collection, etc. 
+
+Le PIM utilise le nom du fichier pour savoir pour quelle ressource le fichier doit être utilisé et si cette ressource est localisée ou pas. 
+
+**Exemple 1 avec une ressource non-localisable**
```

Comment:
```suggestion
**Exemple 1 avec une ressource non-localisable**  
```
You need to add two spaces at the end of your line if you want your content to actually go to the next line.